include/roboteam_utils/Rectangle.h

```diff
@@ -126,9 +111,40 @@ class Rectangle {
      * @brief Writes a textual representation of this rectangle to the given output stream.
      */
     std::ostream &write(std::ostream &out) const;
-};
 
-std::ostream &operator<<(std::ostream &out, const Rectangle &rect);
+    /* 	 ______   _______  _______  ______     _______  _______  ______   _______
+     *	(  __  \ (  ____ \(  ___  )(  __  \   (  ____ \(  ___  )(  __  \ (  ____ \
+     *	| (  \  )| (    \/| (   ) || (  \  )  | (    \/| (   ) || (  \  )| (    \/
+     *	| |   ) || (__    | (___) || |   ) |  | |      | |   | || |   ) || (__
+     *	| |   | ||  __)   |  ___  || |   | |  | |      | |   | || |   | ||  __)
+     *	| |   ) || (      | (   ) || |   ) |  | |      | |   | || |   ) || (
+     *	| (__/  )| (____/\| )   ( || (__/  )  | (____/\| (___) || (__/  )| (____/\
+     *	(______/ (_______/|/     \|(______/   (_______/(_______)(______/ (_______/
+     *
+     * The functions below are dead. Remove this tag if you use any of the functions and make sure to remove this tag at other places as well that will become alive by using any of the
+     * function below. Do not read/document/redesign/analyse/test/optimize/etc. any of this code, because it is a waste of your time! This code was not removed or placed at another
+     * branch, because other software developers are very attached to this code and are afraid that this code might be used at some day (but I think it won't be used at all and should
+     * be removed).
+     */
```

Comment:
Please no. I really disagree with having big tags in the codebase like this. It results in unnecessary clutter. You can put a -not maintained- tag over it if you must, but this is excessive and feels more like a slight against your team members because they don't agree with you